src/java/org/apache/cassandra/io/sstable/format/big/BigFormat.java

```diff
@@ -151,6 +154,7 @@ public SSTableReader open(SSTableReaderBuilder builder)
             hasCommitLogLowerBound = version.compareTo("mb") >= 0;
             hasCommitLogIntervals = version.compareTo("mc") >= 0;
             hasAccurateMinMax = version.compareTo("md") >= 0;
+            hasOriginatingHostId = version.matches("(m[e-z])|(n[b-z])");
```

Comment:
`o*` versions are good too ;)
Noone's gonna remember to change this if `o` is ever introduced/
I think we need to make some comparisons with `version.compareTo("nb") >= 0` being the last